posthog/hogql/database/argmax.py

```diff
@@ -1,13 +1,16 @@
 from typing import Optional
 from collections.abc import Callable
 
+from posthog.hogql.parser import parse_expr
+
 
 def argmax_select(
     table_name: str,
     select_fields: dict[str, list[str | int]],
     group_fields: list[str],
     argmax_field: str,
     deleted_field: Optional[str] = None,
+    no_future_field: Optional[str] = None,
```

Comment:
Would be useful to document the meaning of this in the function's docstring, as it's not immediately obvious, being very specific to the persons use case

Reply:
renamed to `timestamp_field_to_clamp` for now